### PR TITLE
A: https://www.zhihu.com/

### PR DESCRIPTION
**Pull request by shenzhiming88**
ref: https://github.com/Crystal-RainSlide/AdditionalFiltersCN/blob/80a8d7dcefaef873ba772696611342ab4ba9138b/CN/DC.txt#L240-L245
image

### CODE REVIEW

File: easyprivacy/easyprivacy_specific_international.txt [hunk 1]
```diff
@@ -309,6 +310,7 @@
 ||statistic.takungpao.com^
 ||stats.autohome.com.cn^
 ||stats.yinyuetai.com^
+||sugar.zhihu.com^
```
Comment (spirillen) [inline easyprivacy/easyprivacy_specific_international.txt:313]: Touches https://forums.lanik.us/viewtopic.php?p=151451#p151451

Comment (shenzhiming88) [reply easyprivacy/easyprivacy_specific_international.txt:313]: Hello! The file seems to have been renamed now and disabling the domain does not seem to have any effect

Comment (spirillen) [reply easyprivacy/easyprivacy_specific_international.txt:313]: Current tracking domain is `zhihu-web-analytics.zhihu.com` (https://mypdns.org/my-privacy-dns/matrix/-/issues/1176) but it is still resolving.

Comment (shenzhiming88) [reply easyprivacy/easyprivacy_specific_international.txt:313]: Ok, when I have time I will open another Pull Request and delete this domain, thank you!

Comment (spirillen) [reply easyprivacy/easyprivacy_specific_international.txt:313]: > Ok, when I have time I will open another Pull Request and delete this domain, thank you!

You can just edit the file you have submitted and the re-submit it

Comment (shenzhiming88) [reply easyprivacy/easyprivacy_specific_international.txt:313]: Okay
Then please ask
```
analytics.zhihu.com
zhihu-analytics.zhihu.com
```
Do I need to add these two domains? Thank you

Comment (spirillen) [reply easyprivacy/easyprivacy_specific_international.txt:313]: Ping @ryanbr @Khrin :arrow_double_up:

Comment (ryanbr) [reply easyprivacy/easyprivacy_specific_international.txt]:
```
||unpkg.zhimg.com/za-js-sdk^
||unpkg.zhihu.com/za-js-sdk^  
```
theses don't look like trackers

Comment (shenzhiming88) [reply easyprivacy/easyprivacy_specific_international.txt]: @ryanbr Okay, it has been modified

**Comment by ryanbr** [comment]
`||zhihu.com/zbst/events/` included already in https://github.com/easylist/easylist/commit/f55736d02539b0f8903369bde9cf7509892f0183

**Comment by shenzhiming88** [comment]
> `||zhihu.com/zbst/events/` included already in [f55736d](https://github.com/easylist/easylist/commit/f55736d02539b0f8903369bde9cf7509892f0183)

Hello!
But I didn't add this filter rule

**Comment by ryanbr** [comment]
Remove your commit, we already have this filter.
zhihu

**Comment by shenzhiming88** [comment]
> Remove your commit, we already have this filter.
> zhihu

Hello!
This is for conflict resolution, you can see File changes

**Comment by ryanbr** [comment]
Closing. not resolved.

**Pull request closed without merging**